src/Graph12/Graph12.tsx

```diff
@@ -0,0 +1,53 @@
+import { ProgressCircle } from "../components/ProgressCircle";
+const data = [
+	{
+		label: "MAGNA",
+
+		count: 878,
+	},
+	{
+		label: "VELIT",
+
+		count: 618,
+	},
+	{
+		label: "DOLOR",
+
+		count: 521,
+	},
+	{
+		label: "CUPLA",
+
+		count: 221,
+	},
+];
+const maxData = 1000;
+
+const percentages = data.map((item) =>
+	Math.round((item.count / maxData) * 100)
+);
+
+export function Graph12() {
+	return (
+		<div className="mt-2">
+			<div className="flex items-center gap-3 w-[250px] ml-3">
+				<div className="bg-blue-500 h-3 w-3 rounded-full block"></div>
+				<span className="uppercase text-[0.5rem]">
+					Lorem Ipsum is simply dummy lorem dummy lorem
+				</span>
+			</div>
+			<div className="flex">
+				{percentages.map((percent, index) => (
```

Comment:
I would use data here and calculate % value directly inside ProgressCircle. Less props You need to pass down the better

Reply:
done